Docs for Gloo Federation (#3387)
solo-io/gloo: docs/content/guides/gloo_federation/cluster_registration.md

---
title: Cluster Registration
description: Registering a cluster with Gloo Federation
weight: 20
---

> Gloo Federation monitors clusters that have been registered with the Federation service and automatically discovers instances of Gloo deployed in the registered cluster. Once the registration process is complete, Gloo Federation can create federated configuration resources and apply them to Gloo instances running in registered clusters.

Reviewer: `Federation service`? Just wondering if we could use slightly different language to describe this.

> Gloo Federation monitors clusters that have been registered with the Federation service and automatically discovers instances of Gloo deployed in the registered cluster. Once the registration process is complete, Gloo Federation can create federated configuration resources and apply them to Gloo instances running in registered clusters.

Reviewer: ```suggestion
Gloo Federation monitors clusters that have been registered using `glooctl`, and automatically discovers instances of Gloo deployed on said clusters. Once the registration process is complete, Gloo Federation can create federated configuration resources and apply them to Gloo instances running in registered clusters.
```
